sections/editing.include

```diff
@@ -1667,14 +1668,24 @@
   If specified, the value must be a single printable character:
   typically a string one Unicode code point in length.
   
-  <p class="warning">Character references such as <code>accesskey="&#65"</code> are not valid values, 
-  because they will cause no shortcut to be assigned in many browsers. 
-  However certain printable characters that may represent more than one Unicode code point, such as 
-  <code>accesskey="श्र"</code> are valid.</p>
+  <div class="warning">
+    Composed characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form D</a>,
+    such as <code>accesskey="&amp;#x006E;&amp;#x0303;"</code>
+    to assign <samp>ñ</samp> as a shortcut, are not valid values,
+    and in many browsers no shortcut will be assigned. Characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form C</a>
+    are valid values. Printable characters that may represent more than one Unicode code point, such as 
+    <code>accesskey="श्र"</code>, are valid values.
   
```

Comment:
Yep. If you have two or more characters, IE accepts the first as an accesskey and everything else just stops recognising that there is one. This is part of what was tested a lot that resulted in changing accesskey for HTML 5.1 from the nice but speculative approach that allowed for assigning multiple things.

What I don't recall if I have tried is e.g. a word in a non-latin script... I'll check now.